### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by marcverhagen**
Mostly repository-level documentation and documentation of batches

### CODE REVIEW

File: batches/aapb-collaboration-7-leftover.txt [hunk 1]
```diff
@@ -1,4 +1,13 @@
-# see https://github.com/clamsproject/aapb-annotations/issues/79 for more info
+# --------------------------------------------------------------------------------
+# Originally, a set of about 4000 videos was selected for slate annotation. This 
+# batch has the GUIDs that were not include in batch aapb-collaboration-7, which
+# contains the GUIDs of assets that were annotated.
+# 
+# See https://github.com/clamsproject/aapb-collaboration/issues/79 for more info.
```
Comment (keighrim) [inline batches/aapb-collaboration-7-leftover.txt:6]: looks like there's no issue 79, and issue 7 in the collab repo seems to be the right one.

File: batches/aapb-collaboration-7.txt [hunk 2]
```diff
@@ -1,4 +1,12 @@
-# see https://github.com/clamsproject/aapb-collaboration/issues/7 for more info
+# --------------------------------------------------------------------------------
+# A batch with 887 videos for slate annotation. Originally part of a set of about
+# 4000 videos that were selected for this task.
+# 
+# See https://github.com/clamsproject/aapb-collaboration/issues/79 for more info.
+#
+# Note that the issue identifier on this batch does not match the related issue,
+# probably due to a human error somewhere in the loop.
```
Comment (keighrim) [inline batches/aapb-collaboration-7.txt:8]: this one had the correct issue URL.

File: batches/aapb-collaboration-7-leftover.txt [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
+#
+# Note that the issue identifier on this batch does not match the related issue,
+# probably due to a human error somewhere in the loop.
```
Comment (keighrim) [inline batches/aapb-collaboration-7-leftover.txt:9]: Yeah, definitely was my bad from 16224dbe59e644002c9317a985e8607c8cc5e6fb. Probably a typo...

File: repository_level_conventions.md [hunk 4]
```diff
@@ -9,55 +13,59 @@
...
-If configurable, the annotation tool should be configured to use this format. If possible, the annotator should also be instructed to use this format.  
+The time format for all (gold) datasets in this repository is [ISO 8601 Time Format](https://en.wikipedia.org/wiki/ISO_8601#Times), with time precision of up to hours and down to milliseconds placed after seconds with a **DOT**, where `00:00:00.000` means the very beginning of a time-based media (not midnight real-time). This instruction is for the "gold" data formats throughout this repository, except where exceptions are required and documented.
+
+> _TODO: Current/Old gold datasets and tools have yet to be converted._
+
+During raw annotation however, third-party annotation tools may use different time formats. The expectation for any in-house tools and apps is to use this standard. If configurable, annotations tool should be configured to use this format. If possible, the annotator should also be instructed to use this format.
+
+> _TODO: the following prose is unclear_
+
 For MPEG-based video files, frame numbers are converted to milliseconds with loss of precision past 3-digits.
```
Comment (keighrim) [inline repository_level_conventions.md:32]: `past 3-digits` meaning "below third decimal place"?

File: repository_level_conventions.md [hunk 5]
```diff
@@ -9,55 +13,59 @@
...
+
+> _TODO: Current/Old gold datasets and tools have yet to be converted._
+
+During raw annotation however, third-party annotation tools may use different time formats. The expectation for any in-house tools and apps is to use this standard. If configurable, annotations tool should be configured to use this format. If possible, the annotator should also be instructed to use this format.
+
+> _TODO: the following prose is unclear_
+
 For MPEG-based video files, frame numbers are converted to milliseconds with loss of precision past 3-digits. 
 However, due to exact time -> still-image-fetching being dependent on the video compression/codec/player, there is no expected need for precision past 3-digits. 
 It is assumed that different video players will regenerate images on screen slightly differently based on the decompression algorithms.
```
Comment (keighrim) [inline repository_level_conventions.md:34]: I guess, these two lines could be something like 
"However, due to algorithmic differences in compression/decompression and their implementations in video players for human watchers, losing temporal precision below milliseconds mustn't be a significant problem"

**Review by keighrim — CHANGES_REQUESTED** [review]
Looks great! Thanks for this mass cleaning up of the documents. I left some comments for minor corrections, but nothing major. 

Before adding more commits to `99` branch or merging this PR, I'm a bit concerned about another merge commits from `99` to `dev` branch. (a403742442a87bbef794d74e8ff38ed3a2044f98 and c72edab9a56625402477dd73eaa6d91d68de6160). Looking at the log of the `dev` branch, I think if we merge this PR (`99`> `main`) the only distinct commits left in `dev` is essentially identical to `85` branch, for which a separate PR is already open (https://github.com/clamsproject/aapb-annotations/pull/102). 

How do you want to proceed? The `dev` branch has merged `85` first then merged `99`. So I think we can do one of 
1. conclude and merge #102 first and then merge this PR, then discard `dev` branch
2. merge this PR (to `main`) first and re-do the resolution of conflict to merge #102, then discard `dev` branch. 
3. change this PR to merge to `dev` to close #99 and then later merge `dev` to `main` to close #85. 

Also, I'm not sure how significant `dev` branch is with regard to the separate "dashboarding" effort at https://github.com/clamsproject/dashboard. Unless we really need to keep the `dev` branch for the dashboard, I think the second option would be the least conflicting/complicated way of handling...

File: batches/aapb-collaboration-7-leftover.txt (hunk 1 above)
Comment (marcverhagen) [reply batches/aapb-collaboration-7-leftover.txt:6]: Yup, seems to be a typo on my end.

File: batches/aapb-collaboration-7.txt (hunk 2 above)
Comment (marcverhagen) [reply batches/aapb-collaboration-7.txt:8]: I removed that comment

File: batches/aapb-collaboration-7-leftover.txt (hunk 3 above)
Comment (marcverhagen) [reply batches/aapb-collaboration-7-leftover.txt:9]: Hmmm, I am mixing up my comments between leftover and non-leftover, but in general I think all that was needed is to turn 79 into 7 and remove the note that something was awry. Which I did

**Comment by marcverhagen** [comment]
My only reason for having the develop branch was to have a version where the documentation changes and some other changes were both present, without merging them into the documentation branch. It looks like I never completely followed up on that though. 

Your option 1 above is the one that I would choose.

> Also, I'm not sure how significant `dev` branch is with regard to the separate "dashboarding" effort at https://github.com/clamsproject/dashboard. Unless we really need to keep the `dev` branch for the dashboard, I think the second option would be the least conflicting/complicated way of handling... 

It is there so I can make changes without the github pages immediately being updated.

But in general, there are interactions between the the dashboard, annotations and evaluations repositories and I am open to discussion on how to best manage those.

**Review by keighrim — APPROVED** [review]

**Comment by keighrim** [comment]
Although #102 isn't yet merged, I found the PR and the underlying issue (https://github.com/clamsproject/aapb-annotations/issues/85) need further discussion. So I merging this PR first, and purging the `develop` branch.

**Pull request merged**